### DISCUSSION TITLE
How long does it typically take to review an application?

### DISCUSSION THREAD

**Discussion by PerpetualCreativity** [community/community#139956] [GitHub Education]
### Hi everyone,

I submitted my application in late August, so I just wanted to know how long it typically takes. My applications for previous years were approved within a few days. I have my university email linked, as well. The application form did not ask me for any supporting documents (this is fine if it re-uses documents, since I am at the same university).

**Comment by davevad93** [comment]
Hi @PerpetualCreativity ,

It usually takes a few days and sometimes it can take a little longer, around a couple of weeks that's because the waiting time may vary depending on different factors, such as the volume of requests. However, if you submitted your application at the end of August, it's already been a month and it usually doesn't take that long. I assume your current status is `Congrats! Your request for @PerpetualCreativity was approved and your benefits will be available soon.`

**Reply by PerpetualCreativity** [reply]
This is what I see at the top of education.github.com/pack:
![image](https://github.com/user-attachments/assets/b6f724a9-d4f7-4dbd-862a-c9842118aaa5)

**Reply by davevad93** [reply]
And what's your status on the [application page](https://education.github.com/discount_requests/application) where it says "***You've already submitted X requests***"?

**Reply by PerpetualCreativity** [reply]
I don't see a status on that page. Just "You've already submitted 2 requests".

**Comment by Xelliese** [comment]
If you've linked your uni email, that should help speed things up since they can verify your info easily.

**Reply by PerpetualCreativity** [reply]
I've had my uni email linked for the past two years. It showed up when I applied.
